cfpb/design-manual
# Update for capital-framework 5.3.0

Updates the project to use the latest release from Capital Framework.

## Changes

- Updates cf-forms to 5.2.0

## Testing

- Run `npm start`
- Open `http://localhost:4000/page-components/form-fields.html#notifications` and view the new field level success and warnings, the error should look the same.

## Screenshots

<img width="755" alt="screen shot 2018-07-17 at 12 40 45 pm" src="https://user-images.githubusercontent.com/1280430/42835523-a22945b2-89be-11e8-9adb-16bcc1c4cb54.png">
